api/src/controllers/referent.js

```diff
@@ -358,6 +358,42 @@ router.put("/young/:id", passport.authenticate("referent", { session: false, fai
   }
 });
 
+router.put("/young/change_cohort/:id", passport.authenticate("referent", { session: false, failWithError: true }), async (req, res) => {
+  try {
+    const { error, value } = validateYoung(req.body);
+    if (error) return res.status(400).send({ ok: false, code: ERRORS.INVALID_PARAMS, error: error.message });
+
+    const { id } = req.params;
+    const young = await YoungModel.findById(id);
+    if (!young) return res.status(404).send({ ok: false, code: ERRORS.YOUNG_NOT_FOUND });
+
+    // eslint-disable-next-line no-unused-vars
+    let { __v, ...newYoung } = value;
+
+    const oldSessionPhase1Id = young.sessionPhase1Id;
+    if (young.cohort !== req.body.cohort && (young.sessionPhase1Id || young.meetingPointId)) {
+      newYoung.sessionPhase1Id = "";
+      newYoung.meetingPointId = "";
```

Comment:
en mongo, si tu set un champs a undefined, ca le supprime, ca change pas grand chose mais c'est clean
```suggestion
      young.set({ sessionPhase1Id: undefined} );
      young.set({ meetingPointId: undefined} );
```